geode-core/src/main/java/org/apache/geode/management/internal/MBeanJMXAdapter.java

```diff
@@ -71,14 +69,17 @@
 
   private DistributedMember distMember;
 
-  private Logger logger = LogService.getLogger();
+  private Logger logger;
 
   /**
    * public constructor
    */
-  public MBeanJMXAdapter() {
-    this.localGemFireMBean = new ConcurrentHashMap<>();
-    this.distMember = InternalDistributedSystem.getConnectedInstance().getDistributedMember();
+  public MBeanJMXAdapter(
+      Map<ObjectName, Object> localGemFireMBean,
+      DistributedMember distMember, Logger logger) {
+    this.localGemFireMBean = localGemFireMBean;
+    this.distMember = distMember;
+    this.logger = logger;
```

Comment:
you probably can get away with only injecting the DistributedMember for this constructor to satisfy all your mocking need in the junit test